File: _scratch/run/removing_stars_from_a_string/sol.py
class Solution:
    def removeStars(self, s):
        st = []
        for c in s:
            if c == "*":
                st.pop()
            else:
                st.append(c)
        return "".join(st)
import sys
_l=sys.stdin.read().split('\n')
_T=int(_l[0]);_o=[]
for _i in range(_T):
    _s=_l[1+_i] if 1+_i<len(_l) else ""
    _o.append(str(Solution().removeStars(_s)))
print('\n'.join(_o))

File: _scratch/run/removing_stars_from_a_string/sol.js
/**
 * @param {string} s
 * @return {string}
 */
var removeStars = function(s) {
    const st = [];
    for (const c of s) {
        if (c === '*') st.pop();
        else st.push(c);
    }
    return st.join('');
};
const _l=require('fs').readFileSync(0,'utf8').split('\n');const _T=+_l[0];const _o=[];
for(let _i=0;_i<_T;_i++){const _s=(_l[1+_i]||'').replace(/\r$/,'');_o.push(String(removeStars(_s)));}
console.log(_o.join('\n'));